spec/lib/oapi_tasks_spec.rb

```diff
@@ -115,7 +119,7 @@ class Base < Grape::API
         end
 
         it 'returns complete doc' do
-          expect(response['paths'].length).to eql 2
+          expect(response['paths'].length).to eql 3
```

Comment:
We added my_doc to check that paths containing "doc" are not targeted by urls_for.
Consequently, the number of tests here has also changed.